src/routes/Register/container.js

```diff
@@ -46,6 +50,16 @@ export class RegisterContainer extends Component {
 		}
 	}
 
+	componentDidUpdate(prevProps) {
+		const { user: prevUser } = prevProps;
+		const { user } = this.props;
+		const { registered } = this.state;
+
+		if (!registered && !prevUser && user) {
```

Comment:
Do we really need `this.state.registered`?
I mean, if the user is on Register view and doesn't have a `prevUser` yet, he's not registered.
Then it would be just a matter of testing `prevUser` against `user`.

Everything else LTGM.

Reply:
@diegolmello I have made other improvements on the codebase, so yeah, we don't need the state anymore.
Thanks.